applications/StructuralMechanicsApplication/tests/test_constitutive_law.py

```diff
@@ -435,7 +429,6 @@ def _generic_constitutive_law_test(model_part, deformation_test):
                 # Chauchy
                 cl.CalculateMaterialResponseCauchy(cl_params)
                 cl.FinalizeMaterialResponseCauchy(cl_params)
-                cl.FinalizeSolutionStep(properties, geom, N, model_part.ProcessInfo)
```

Comment:
you can just omit the call?

Reply:
Yes, basically right now no law (except plastic truss law) is up to date respect to this